index.js

```diff
@@ -2,13 +2,30 @@ const express = require('express');
 const helmet = require('helmet');
 const cors = require('cors');
 
+const usersRoutes = require('./users/usersRoutes');
+
 const server = express();
 
 server.use(helmet());
 server.use(cors());
 server.use(express.json());
 server.use(express.urlencoded({ extended: true }));
 
+server.use((req, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader(
+    'Access-Controll-Allow-Methods',
+    'OPTIONS, GET, POST, PUT, PATCH, DELETE',
+  );
+  res.setHeader(
+    'Access-Control-Allow-Headers',
+    'Content-Type, Authorization',
+  );
+  next();
+});
```

Comment:
The cors() package has already taken care of ```Cross-Origin Request Blocked: The Same Origin Policy```, and removes the stress of manually setting header.  You should have either one, but not both.